tests/uniswapTrades.test.ts

```diff
@@ -1,29 +1,19 @@
 import { expect } from 'chai'
 import JSBI from 'jsbi'
-import { ethers, utils, Wallet } from 'ethers'
+import { utils, Wallet } from 'ethers'
 import { expandTo18Decimals } from '../src/utils/expandTo18Decimals'
-import { SwapRouter } from '../src'
-import { MixedRouteTrade, MixedRouteSDK, Trade as RouterTrade } from '@uniswap/router-sdk'
+import { SwapRouter, UniswapTrade } from '../src'
+import { MixedRouteTrade, MixedRouteSDK } from '@uniswap/router-sdk'
 import { Trade as V2Trade, Pair, Route as RouteV2 } from '@uniswap/v2-sdk'
-import { Trade as V3Trade, Route as RouteV3, Pool, FeeAmount } from '@uniswap/v3-sdk'
-import { SwapOptions } from '../src'
-import { PermitSingle } from '@uniswap/permit2-sdk'
+import { Trade as V3Trade, Route as RouteV3, Pool } from '@uniswap/v3-sdk'
 import { generatePermitSignature, toInputPermit } from './utils/permit2'
-import { ROUTER_ADDRESS } from './utils/addresses'
-import { CurrencyAmount, TradeType, Ether, Token, Percent, Currency } from '@uniswap/sdk-core'
+import { CurrencyAmount, TradeType } from '@uniswap/sdk-core'
 import { registerFixture } from './forge/writeInterop'
-import { getPair, getPool } from './utils/uniswapData'
+import { buildTrade, makePermit, getUniswapPools, swapOptions, ETHER, DAI, USDC } from './utils/uniswapData'
 import { hexToDecimalString } from './utils/hexToDecimalString'
 
-const ETHER = Ether.onChain(1)
-const RECIPIENT = '0xaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa'
-const WETH = new Token(1, '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2', 18, 'WETH', 'Wrapped Ether')
-const DAI = new Token(1, '0x6B175474E89094C44Da98b954EedeAC495271d0F', 18, 'DAI', 'dai')
-const USDC = new Token(1, '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48', 6, 'USDC', 'USD Coin')
-const FEE_AMOUNT = FeeAmount.MEDIUM
 const FORK_BLOCK = 16075500
```

Comment:
Dont actually need to have this given that `getUniswapPools` has this value as its default block